subprojects/docs/src/docs/userguide/dep-man/06-publishing/publishing_maven.adoc

```diff
@@ -188,6 +188,54 @@ For integration with a local Maven installation, it is sometimes useful to publi
 
 The Maven Publish Plugin makes this easy to do by automatically creating a link:{javadocPath}/org/gradle/api/publish/maven/tasks/PublishToMavenLocal.html[PublishToMavenLocal] task for each link:{groovyDslPath}/org.gradle.api.publish.maven.MavenPublication.html[MavenPublication] in the `publishing.publications` container. The task name follows the pattern of `publish__PubName__PublicationToMavenLocal`. Each of these tasks is wired into the `publishToMavenLocal` aggregate task. You do not need to have `mavenLocal()` in your `publishing.repositories` section.
 
+[[publishing_maven:relocation]]
+== Publishing Maven relocation information
+
+When a project changes the `groupId` or `artifactId` (the _coordinates_) of an artifact it publishes, it is important to let users know where the new artifact can be found. Maven can help with that through the _relocation_ feature. The way this works is that a project publishes an additional artifact under the old coordinates consisting only of a minimal _relocation POM_; that POM file specifies where the new artifact can be found. Maven repository browsers and build tools can then inform the user that the coordinates of an artifact have changed.
```

Comment:
I have intentionally not added a link to https://maven.apache.org/guides/mini/guide-relocation.html here because that guide mentions "Maven 2" (though I assume it works for Maven 3 as well) and has a "2020 rework in progress" note at the beginning.